### PR TITLE
Release 19 branch lts deprecation

### PR DESCRIPTION
**Pull request by miguelalonsojr**
### Proposed change(s)

Deprecated support for Unity version 2019.4. Supported version is now 2020.3.

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [X] Other (please describe) LTS support updated to 2020.3.

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [X] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/main/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [X] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/main/docs) (if applicable)
- [X] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Migrating.md) (if applicable)

### Other comments
Updated yamato CI to support Unity v2020.3+.

### CODE REVIEW

File: Project/Packages/manifest.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
   "dependencies": {
     "com.unity.ide.rider": "3.0.5",
+    "com.unity.ide.visualstudio": "2.0.12",
```
Comment (cdettmering-unity) [inline Project/Packages/manifest.json:4]: Why was this added?

**Review by cdettmering-unity — COMMENTED** [review]
Overall, looks fine syntactically, as long as all the tests pass, I'll be ready to approve.

**Review by cdettmering-unity — APPROVED** [review]

**Comment by zhkj** [comment]
hi.  I notice that Unity version 2019.4 is not longer supported in Release 19 branch. Could you explain why? What new features are only supported in higher Unity version?
